From 722923bc9b788f028e4221c77138fc66735a3113 Mon Sep 17 00:00:00 2001
From: Petr Sumbera <petr.sumbera@oracle.com>
Date: Fri, 3 May 2013 22:25:38 +0800
Subject: [PATCH] Fix for compilation with python 2.6.8

Remap PyCapsule to PyCObject based on the macro being available
instead of using the python version
---
 python/libxml_wrap.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/python/libxml_wrap.h b/python/libxml_wrap.h
index a9b9739..53a0618 100644
--- a/python/libxml_wrap.h
+++ b/python/libxml_wrap.h
@@ -34,11 +34,11 @@
 #define PyBytes_AsString PyString_AsString
 #define PyBytes_AS_STRING PyString_AS_STRING
 #define PyBytes_GET_SIZE PyString_GET_SIZE
-
+#endif
+#ifndef PyCapsule_New
 #define PyCapsule_New PyCObject_FromVoidPtrAndDesc
 #define PyCapsule_CheckExact PyCObject_Check
 #define PyCapsule_GetPointer(o, n) PyCObject_GetDesc((o))
-
 #endif
 #endif
 
-- 
1.8.3.2

